pkgs/games/wireworld/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, zip
+, love
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+stdenv.mkDerivation rec {
+  pname = "Wireworld";
```

Comment:
```suggestion
  pname = "wireworld";
```
I'm not sure about this but I haven't seen any package names with uppercase letters. Maybe we should normalize it here

Reply:
yeah you right, I forgot to make it lowercase. Will change that.